lib/src/components/NxGlobalSidebar/NxGlobalSidebar.tsx

```diff
@@ -55,7 +56,7 @@ const NxGlobalSidebar: FunctionComponent<Props> = function NxGlobalSidebar(props
   return (
     <div className={classes} id={id}>
       <aside aria-label="global sidebar">
-        <div className="nx-global-sidebar__header">
+        <div className="nx-global-sidebar__header" ref={headerRef} style={{height: headerRef?.current?.clientHeight}}>
```

Comment:
This was the solution I came up with to keep the header height the same whether the sidebar is open or closed to make sure the nav items don't shift up when closed. However, the height won't update if the sidebar defaults to closed and then opens with a logo image taller than 34px. Are there scenarios in product where the sidebar is closed by default?

Reply:
> This was the solution I came up with to keep the header height the same whether the sidebar is open or closed to make sure the nav items don't shift up when closed. However, the height won't update if the sidebar defaults to closed and then opens with a logo image taller than 34px. Are there scenarios in product where the sidebar is closed by default?

It should be possible to set the width of the `.nx-global-sidebar__product-info` to 0 when collapsed, rather than using `display: none`.  That way its height will still be part of the layout.

Reply:
> > This was the solution I came up with to keep the header height the same whether the sidebar is open or closed to make sure the nav items don't shift up when closed. However, the height won't update if the sidebar defaults to closed and then opens with a logo image taller than 34px. Are there scenarios in product where the sidebar is closed by default?
> 
> It should be possible to set the width of the `.nx-global-sidebar__product-info` to 0 when collapsed, rather than using `display: none`. That way its height will still be part of the layout.

I initially tried changing the width, and found an interesting problem that I couldn't figure out why it was happening. Setting the width to 0px works for the example with the 43px tall logo, but in the 34px tall logo, `.nx-global-sidebar__product-info` increased to 40px tall (while the logo img remained at 34px tall)

Reply:
> I initially tried changing the width, and found an interesting problem that I couldn't figure out why it was happening. Setting the width to 0px works for the example with the 43px tall logo, but in the 34px tall logo, .nx-global-sidebar__product-info increased to 40px tall (while the logo img remained at 34px tall)

Hmm that's not happening for me.  See https://github.com/sonatype/sonatype-react-shared-components/pull/1090

Reply:
> > I initially tried changing the width, and found an interesting problem that I couldn't figure out why it was happening. Setting the width to 0px works for the example with the 43px tall logo, but in the 34px tall logo, .nx-global-sidebar__product-info increased to 40px tall (while the logo img remained at 34px tall)
> 
> Hmm that's not happening for me. See #1090

Merged #1090 here [c98e395](https://github.com/sonatype/sonatype-react-shared-components/pull/1078/commits/c98e395d3a815eb269a2bd777b4c1bea37226d26). Looking through it I found where my initial CSS mistake was when overriding the `display:none` set on `.nx-global-sidebar__expanded-content`. Thanks for the fix!